src/components/SearchList.js

```diff
@@ -1,14 +1,20 @@
-import React, { useState } from "react";
+import React, { useState, useEffect } from "react";
 
 const SearchList = () => {
```

Comment:
The ticket indicates:

```
Props this component can receive:

A list of search result albums
```

So this should take some props; can so something like: 

```suggestion
const SearchList = ({ albumResults }) => {
```